src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -63,6 +63,39 @@ public class DatabaseMetaDataTest extends AbstractTest {
     private static final String functionName = RandomUtil.getIdentifier("DBMetadataFunction");
     private static Map<Integer, String> getColumnsDWColumns = null;
     private static Map<Integer, String> getImportedKeysDWColumns = null;
+    private static Map<String, Class<?>> getColumnMetaDataClass = null;
```

Comment:
this is only ever used in the 1 new test, doesn't need to be in the class scope

Reply:
Done